user/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+class User(models.Model):
+    email         = models.EmailField(max_length = 200, unique = True)
+    password      = models.CharField(max_length = 500)
+    phone         = models.CharField(max_length = 45, unique = True)
+    birth         = models.DateField()
+    name          = models.CharField(max_length = 45, unique = True)
+    gender        = models.ForeignKey('Gender', on_delete = models.SET_NULL, null = True)
+    alarm_confirm = models.BooleanField()
+    created_at    = models.DateTimeField(auto_now_add = True)
+    updated_at    = models.DateTimeField(auto_now = True)
+
+    class Meta:
+        db_table = 'users'
+
+class Gender(models.Model):
+    name = models.CharField(max_length = 45)
+
+    class Meta:
+        db_table = 'users'
```

Comment:
아래 테이블명 확인 부탁드립니다. 젠더 name은 max_length 10정도로 줘도 될것같아요.
```suggestion
        db_table = 'genders'
```